### PR TITLE
404 Page

### PR DESCRIPTION
**Pull request by KarthikSiddarth**
# Resolves

- [AR-545](https://team-1624093970686.atlassian.net/browse/AR-545)

## Changes

Continued work of @shrinathprabhu, removed description, aligned image and text as per [design](https://www.figma.com/file/WTruG74nclNspjP1rwSbzP/Website-VD-Exploration?node-id=2556%3A221)

# Screenshots
Screenshot 2022-02-08 at 6 19 38 PM

# Checklist

- [X] The branch name follows the format: `developer-name/AR-XXX-issue-name`.
- [X] The changes have been tested locally.

### CODE REVIEW

**Review by shrinathprabhu — APPROVED** [review]

File: components/PageNotFound.vue [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        md-direction="column-reverse"
+        align="center"
+        md-justify="space-evenly"
+        class="error-content-container"
+      >
+        <v-image
+          path="images/404-illustration.png"
+          class="error-illustration"
+        />
+        <v-stack direction="column" class="error-content">
```
Comment (ajithranka) [inline components/PageNotFound.vue:15]: This wrapping `v-stack` may not be needed as there's only one child element inside.

File: components/PageNotFound.vue [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      </v-stack>
+    </v-container>
+  </section>
+</template>
+
+<style lang="postcss" scoped>
+.error-content-container {
+  height: 80vh;
+  display: flex;
+  flex-direction: column;
```
Comment (ajithranka) [inline components/PageNotFound.vue:27]: The `display: flex` property is set by the `v-stack` implicitly and we can set `flex-direction: column` with a `direction="column"` prop. 

Check if this works. The `md-direction` and `md-justify` props are not required.

```
<v-stack direction="column" align="center" justify="space-between" class="...">
  ...
</v-stack>
```
```

**Review by ajithranka — CHANGES_REQUESTED** [review]

File: components/PageNotFound.vue (hunk 2 above)
Comment (KarthikSiddarth) [reply components/PageNotFound.vue]: done [commit](https://github.com/arcana-network/website/pull/85/commits/49160c8a55fa3b8aeb21061e4e50cb6e2a66f1ef)

File: components/PageNotFound.vue (hunk 1 above)
Comment (KarthikSiddarth) [reply components/PageNotFound.vue]: done. [commit](https://github.com/arcana-network/website/pull/85/commits/adc70e73438069953bad10e762ec4bf064d006a3)

File: components/PageNotFound.vue (hunk 2 above)
Comment (ajithranka) [reply components/PageNotFound.vue]: Can we also remove `md-direction` and `md-justify`?

Comment (KarthikSiddarth) [reply components/PageNotFound.vue]: sorry, missed that. done. [commit link](https://github.com/arcana-network/website/pull/85/commits/bd9447f16b7bd6955ba37d3d8817aabf29159819)

**Review by ajithranka — APPROVED** [review]

File: components/PageNotFound.vue (hunk 2 above)
Comment (KarthikSiddarth) [reply components/PageNotFound.vue]: this [commit](https://github.com/arcana-network/website/pull/85/commits/c0f7fce3a07e6d7479ee90f190b3bf0fc7e79dd7) is needed to align element on mobile screens

### screenshot:
Screenshot 2022-02-09 at 1 06 24 PM

**Pull request merged**